tests/introspection/iOS/iOSApiProtocolTest.cs

```diff
@@ -61,6 +61,15 @@ protected override bool Skip (Type type)
 					return true;
 
 				break;
+			case "MTLFence":
+			case "MTLHeap":
+				if (Runtime.Arch != Arch.DEVICE)
```

Comment:
mmm not sure about the following conditions I think you want to check device only and only if iOS 10 (and friends) but if my mind is not cheating me at this hour this type will get checked if executed in any iOS version as long it is running on device.